applications/schedview-snapshot/values-usdfint.yaml

```diff
@@ -1,3 +1,11 @@
 image:
   # -- Overrides the image tag whose default is the chart appVersion.
   tag: "v0.12.0"
+
+resources:
+  limits:
+    cpu: 2.0
+    memory: "8Gi"
+  requests:
+    cpu: 1.0
+    memory: "4Gi"
```

Comment:
In general I recommend putting this into `values.yaml` instead, since resource consumption is usually fairly similar across environments, and only overriding it in the per-environments file if there's something unusual about that environment.